### ISSUE TITLE
Add ability to skip preview decoding for reply attach media (especially webms since they are very slow to extract frames from)

### ISSUE THREAD

**Issue by KurobaExReports** [KurobaExReports/Reports#671]
It really does take some time with this webm. Canceling doesn't skip it and the preview ends up with an error message anyway if you wait. The only way I found to skip the check is by restarting the app, since canceling leaves you with no file. Can you put a skip button there? And why this happens in the first place?
Here's the webm btw, it's the OP.
https://boards.4chan.org/g/thread/88647903

**Comment by K1rakishou** [comment]
Yeah, some webms may take 10+ seconds for a single frame retrieval. Unfortunately not much I can do here since I'm using the system API for that and if it's slow then it is just what it is. No other way than to use ffmpeg which will increase apk size by 30 megabytes.
